docs/auth/README.md

```diff
@@ -135,19 +137,36 @@ curl -H "Authorization: Bearer $UMS_TOKEN"
 </template>
 </CodeSwitcher>
 
+::: warning
+Keep in mind at this point in time only JS SDK is automatically refresh access token. If you use REST API or Python SDK you need to handle this situation by your own. Token live time 2h 
```

Comment:
Maybe formulate this a bit different to: 
Keep in mind at this point in time only JS SDK automatically refreshes the access token. If you use REST API or Python SDK you need to handle this yourself. The token does have a duration length of 2h

Reply:
it is by purpose to have 2 times. I think Bailey adjusted this part as well